doc/sphinx/language/core/sections.rst

```diff
@@ -66,12 +66,20 @@ usable outside the section as shown in this :ref:`example <section_local_declara
    In this case, the proof should be terminated with :cmd:`Defined` in order to define a constant
    for which the computational behavior is relevant.  See :ref:`proof-editing-mode`.
 
+.. attr:: clearbody
+
+   Using this attribute with a :cmd:`Let` in a section will cause the
+   body of the definition to be cleared in the proof context of following proofs.
+
 .. warn:: @ident is declared opaque but this is not fully respected inside the section and not at all outside the section.
 
-   Terminating the proof for a :cmd:`Let` with :cmd:`Qed` with `Let` is not recommended.
+   Terminating the proof for a :cmd:`Let` with :cmd:`Qed` is not recommended.
    The variable will appear to have no body inside proofs but the kernel will still use
-   the body when checking, and it will be treated the same as a
-   :cmd:`Defined` `Let` when the section is closed.
+   the body when checking.  When the section is closed it will be treated the as if
+   :cmd:`Defined` had been specified.
+
+   This is the same behavior as :attr:`clearbody` so if it is what you
+   intend use the attribute instead.
```

Comment:
Then, should we go beyond warning, and actually deprecate `Let ... Qed.` in favor of `#[ clearbody ] Let ... Defined.`?